api/src/jmh/java/io/opentelemetry/trace/DefaultTracerBenchmarks.java

```diff
@@ -34,7 +35,7 @@
   @Warmup(iterations = 5, time = 1)
   public void measureFullSpanLifecycle() {
     span = tracer.spanBuilder("span").startSpan();
-    try (io.opentelemetry.context.Scope ignored = TracingContextUtils.currentContextWith(span)) {
+    try (io.opentelemetry.context.Scope ignored = Context.current().with(span).makeCurrent()) {
```

Comment:
would it be possible to make this also a method on the span instance?

I'm struggling with what the name of that method would be right now, though.
`span.activate()` ?

Reply:
Oh yeah did have that idea - not sure if it's worth it. But I would just use the same name as `Context`, so `span.makeCurrent()`. Does it seem useful to have?

Reply:
or maybe `Context.inCurrentScope(span)` ?

Reply:
I believe that looks like

```java
interface Context
  static Scope inCurrentScope(ImplicitContextKeyed value) {
    return Context.current().with(value).makeCurrent();
  }
```

For symmetry, I think we would need a version that also accepts `ContextKey`. I think I like this a bit less than adding to `Span` itself (rather, I could add it to `ImplicitContextKeyed`).